versions.cfg

```diff
@@ -25,4 +25,7 @@ cffi = 1.14.4
 
 # requirement for json widget tests to pass
 plone.schema = 1.3.0
-plone.dexterity = 2.9.8
\ No newline at end of file
+plone.dexterity = 2.9.8
+
+# recent pyjwt 
+pyjwt = 2.1.0
```

Comment:
I would put a comment here to remove this pin after release Plone version with coredev PR.

Reply:
It is needed for Plone 5.2 - which is still supported with Py3 here at master.

Reply:
You won't update the version in the 5.2 branch of coredev? At least for Python 3?

Reply:
Well, with conditional version sections this would be technically possible. 

Since the 8.x series is not part of the release of Plone 5.2 (see https://github.com/plone/buildout.coredev/blob/5.2/versions.cfg#L197 and https://github.com/plone/buildout.coredev/blob/5.2/sources.cfg#L118, it uses the 7.x) I do not think this is possible. 

We could add a note to the README which versions are preconditions are needed for use of the 8 series in Plone 5.2.

Reply:
Anyway, it's weird to say that version 8.x is compatible with Plone 5.2 and it will not be tested on coredev. If any unreleased changes in any package, break the version 8.x, we won't know about it until the Plone version is released.

Reply:
Therefore there are the GH Actions test I think. It works against the latest release, I think this is good enough?

Reply:
@mauritsvanrees would it be possible to pin pyjwt to 2.1.0 in the next Plone 5.2.6 release?

Reply:
@tisto We could pin PyJWT to 2.1.0 for Python 3 in Plone 5.2.
But I think plone.restapi 7.x.x needs a small change then.
It uses the deprecated `verify` parameter, which [is now dropped](https://pyjwt.readthedocs.io/en/stable/changelog.html#dropped-deprecated-verify-param-in-jwt-decode).

Reply:
Ah, that is exactly what this PR fixes. Except that it should also happen on 7.x.x, but not the `setup.py` change (or that part only for Python 3).

Reply:
PyJWT 2.x is Python 3 only, so not an option for 5.2.

Keep it as is for Plone 5.2 in the release. Old PyJWT 1.x plus plone.restapi 7.x - it works.

If one wants to use next plone.restapi 8 in Plone 5.2 buildout fails with a message about PyJWT 2 version conflict. I'll add this to the README->installation docs to pin pyJWT to >=2.1.0.

Reply:
Done

Reply:
@jensens @mauritsvanrees I am wondering if we should defer this to the next major release of plone.restapi (9.x). Once plone.volto has seen a first release, we will need to move some things around (e.g. the block behavior needs to move from restapi to plone.volto). The plan is to have stuff ready for the conference, so it won't take so long. What do you think?

Reply:
It looks like the new code from this PR also works with the old PyJWT. At least the tests pass.
So I think we can include this code even in 7.x.x. Just don't touch `setup.py`.
On PY3, `token.decode("utf-8")` should still be done, but only if the token is bytes, which PyJWT 1.7.1 gives us.

So that is one option: be a bit smarter in the code so we can support both versions of PyJWT.

Other option is to only merge the current PR when `plone.restapi` master only targets PY 3, which seems to be what Timo is saying.

I don't really mind either way.

Reply:
I rebased the branch.